people/model.go

```diff
@@ -5,12 +5,17 @@ import "sort"
 type SortedIdentifiers []identifier
 
 type person struct {
-	UUID        string       `json:"uuid"`
-	BirthYear   int          `json:"birthYear,omitempty"`
-	Identifiers []identifier `json:"identifiers,omitempty"`
-	Name        string       `json:"name,omitempty"`
-	Salutation  string       `json:"salutation,omitempty"`
-	Aliases     []string     `json:"aliases,omitempty"`
+	UUID           string       `json:"uuid"`
+	BirthYear      int          `json:"birthYear,omitempty"`
+	Identifiers    []identifier `json:"identifiers,omitempty"`
+	Name           string       `json:"name,omitempty"`
+	Salutation     string       `json:"salutation,omitempty"`
+	Aliases        []string     `json:"aliases,omitempty"`
+	EmailAddress   string       `json:"emailAddress,omitempty"`
+	TwitterHandle  string       `json:"twitterHandler,omitempty"`
```

Comment:
Dammit - I have real typing issues with 'handle'. All those years of enterprise Java...